spec/lib/fields/belongs_to_spec.rb

```diff
@@ -11,7 +11,7 @@
     it "returns a partial based on the page being rendered" do
       page = :show
       owner = double
-      field = Administrate::Field::BelongsTo.new(:owner, owner, page)
+      field = Administrate::Field::BelongsTo.new(Class.new.new,:owner, owner, page)
```

Comment:
In other places I inserted nil.  It doesn't really matter what goes in that slot, because no existing form field types actually use it.  I'm 100% open to other architectures, method signatures, whatever, I just want to be able to get to the resource from the fields.